### PR TITLE
add support for persistentAnchors to XRSession

### PR DESCRIPTION
**Pull request by cabanier**
***
<a href="https://pr-preview.s3.amazonaws.com/cabanier/anchors/pull/80.html" title="Last updated on Nov 18, 2022, 6:11 PM UTC (4c95ee9)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/immersive-web/anchors/80/d8ac12b...cabanier:4c95ee9.html" title="Last updated on Nov 18, 2022, 6:11 PM UTC (4c95ee9)">Diff</a>

### CODE REVIEW

**Comment by cabanier** [comment]
rasied from https://github.com/immersive-web/real-world-geometry/issues/37

**Comment by cabanier** [comment]
@bialpio it seems we forgot to merge this. Can you approve?

File: index.bs [hunk 1]
```diff
@@ -244,6 +246,10 @@ The {{XRFrame/createAnchor(pose, space)}} method, when invoked on an {{XRFrame}}
 
 Note: It is the responsibility of user agents to ensure that the physical origin tracked by the anchor returned by each {{XRFrame/createAnchor(pose, space)}} call aligns as closely as possible with the physical location of <var ignore=''>pose</var> within <var ignore=''>space</var> at the time represented by the frame on which the method is called. Specifically, this means that for spaces that are dynamically changing, user agents should attempt to capture the native origin of such spaces at the app's specified time. This text is non-normative, but expresses the intent of the specification author(s) and contributors and thus it is highly recommended that it is followed by the implementations to ensure consistent behavior across different vendors.
 
+<div class="algorithm unstable">
+{{XRSession/persistentAnchors}} returns a list of the known persistent anchors. This list MUST reflect the keys of the [=XRSession/map of persistent anchors=].
```
Comment (bialpio) [inline index.bs:250]: Did we already specify how this map is initially populated (on session creation)?

**Review by bialpio — APPROVED** [review]
Approving since we mark the new attribute as unstable but we need to start putting some text that describes how this is initialized / when is it cleared up / etc.

**Comment by cabanier** [comment]
> Approving since we mark the new attribute as unstable but we need to start putting some text that describes how this is initialized / when is it cleared up / etc.

Since it reflects the internal map and its behavior is specified, isn't that enough?

**Comment by bialpio** [comment]
> > Approving since we mark the new attribute as unstable but we need to start putting some text that describes how this is initialized / when is it cleared up / etc.
> 
> Since it reflects the internal map and its behavior is specified, isn't that enough?

IIRC we were chatting about specifying how the map is persisted across sessions (and what is the key by which it's persisted - is it per origin?) and when does it get cleared up (i.e. when clearing up history for the origin)? I don't recall what was the final outcome, but IIUC the way the spec is written right now, nothing will end up being persisted across sessions (I'm assuming that the session's map of persistent anchors always starts empty, because we don't say anywhere that it doesn't).

**Pull request merged**